tests/nosetests/pyanaconda_tests/kernel_test.py

```diff
@@ -27,7 +27,8 @@ def value_retrieval_test(self):
         """KernelArguments value retrieval test."""
 
         ka = KernelArguments.from_string(
-            "blah foo=anything bar=1 baz=0 nowhere nothing=indeed beep=off derp=no nobody=0")
+            "inst.blah foo=anything inst.bar=1 baz=0 nowhere inst.nothing=indeed beep=off "
+            "derp=no nobody=0")
```

Comment:
Any reason to change this if you're changing neither the code nor the test?

Reply:
Because it's more appropriate. Before the change we won't test that the prefix is correctly removed.